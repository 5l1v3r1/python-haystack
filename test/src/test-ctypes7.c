#include <stdio.h>
#include <stdlib.h>
#include <unistd.h>

// TEST Arrays

struct ArrayTest1 {
  unsigned int array1[16];
  void * ptr2;
  char char3;
  unsigned int array4[16];
  void * ptr5;
};


int test1(){
  struct ArrayTest1 * node;
  node = (struct ArrayTest1 *) malloc(sizeof(struct ArrayTest1));
  node->ptr2 = node;
  node->ptr5 = node;
  node->char3 = 'X';

  printf("o: test1 %p\n", node);

  return 0;
}


int main(){

  // TEST
  test1();

  printf("pid %u\n",getpid());
  fflush(stdout);
  sleep(-1);

  return 0;
}


