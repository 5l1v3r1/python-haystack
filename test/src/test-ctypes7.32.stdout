o: test1 0x8c89008
pid 1293
